src/validation/providers/validation-error-factory.ts

```diff
@@ -25,7 +32,7 @@ export class ValidationErrorFactory {
       message,
     }));
 
-    winston.error({ ...errPayload, detail });
+    this.loggingService.info({ ...errPayload, detail });
```

Comment:
We should probably keep this as an `error` for now. If somehow the contract with the other services breaks this means that we won't be able to serve the specified resource (either it's an issue on our side or an issue on the services that we consume but should be tackled as soon as possible).

This is a different scenario than clients sending malformed payloads. For the malformed client payloads we can debate if we should consider it an error (or assign it a different level).

Reply:
Agree 100%, I'll change this, it shouldn't happen in a typical scenario, which is totally different from clients sending malformed payloads as you said (in that case I'd support not logging them as `error`).

Reply:
Fixed in [125db74](https://github.com/safe-global/safe-client-gateway-nest/pull/310/commits/125db74a23085f0e92d517551d45c34d48ed4186)